frontend/src/app/shared/components/grids/widgets/widgets.service.ts

```diff
@@ -1,18 +1,248 @@
+        },
+      },
+      {
+        identifier: 'work_packages_calendar',
+        component: WidgetWpCalendarComponent,
+        title: this.I18n.t('js.grid.widgets.work_packages_calendar.title'),
+        properties: {
+          name: this.I18n.t('js.grid.widgets.work_packages_calendar.title'),
+        },
+      },
+      {
+        identifier: 'work_packages_overview',
+        component: WidgetWpOverviewComponent,
+        title: this.I18n.t('js.grid.widgets.work_packages_overview.title'),
+        properties: {
+          name: this.I18n.t('js.grid.widgets.work_packages_overview.title'),
+        },
+      },
+      {
+        identifier: 'time_entries_current_user',
+        component: WidgetTimeEntriesCurrentUserComponent,
+        title: this.I18n.t('js.grid.widgets.time_entries_current_user.title'),
+        properties: {
+          name: this.I18n.t('js.grid.widgets.time_entries_current_user.title'),
+          days: [true, true, true, true, true, true, true],
+        },
+      },
+      {
+        identifier: 'time_entries_list',
+        component: WidgetTimeEntriesProjectComponent,
+        title: this.I18n.t('js.grid.widgets.time_entries_list.title'),
+        properties: {
+          name: this.I18n.t('js.grid.widgets.time_entries_list.title'),
+        },
+      },
+      {
+        identifier: 'documents',
+        component: WidgetDocumentsComponent,
+        title: this.I18n.t('js.grid.widgets.documents.title'),
+        properties: {
+          name: this.I18n.t('js.grid.widgets.documents.title'),
+        },
+      },
+      {
+        identifier: 'members',
+        component: WidgetMembersComponent,
+        title: this.I18n.t('js.grid.widgets.members.title'),
+        properties: {
+          name: this.I18n.t('js.grid.widgets.members.title'),
+        },
+      },
+      {
+        identifier: 'news',
+        component: WidgetNewsComponent,
+        title: this.I18n.t('js.grid.widgets.news.title'),
+        properties: {
+          name: this.I18n.t('js.grid.widgets.news.title'),
+        },
+      },
+      {
+        identifier: 'project_description',
+        component: WidgetProjectDescriptionComponent,
+        title: this.I18n.t('js.grid.widgets.project_description.title'),
+        properties: {
+          name: this.I18n.t('js.grid.widgets.project_description.title'),
+        },
+      },
+      {
+        identifier: 'custom_text',
+        component: WidgetCustomTextComponent,
+        title: this.I18n.t('js.grid.widgets.custom_text.title'),
+        properties: {
+          name: this.I18n.t('js.grid.widgets.custom_text.title'),
+          text: {
+            raw: '',
+          },
+        },
+      },
+      {
+        identifier: 'project_status',
+        component: WidgetProjectStatusComponent,
+        title: this.I18n.t('js.grid.widgets.project_status.title'),
+        properties: {
+          name: this.I18n.t('js.grid.widgets.project_status.title'),
+        },
+      },
```

Comment:
The `WidgetProjectStatusBetaComponent` got missed somehow (commit 322b7b9a)

```suggestion
      },
      {
         identifier: 'project_status_beta',
         component: WidgetProjectStatusBetaComponent,
         title: i18n.t('js.grid.widgets.project_status_beta.title'),
         properties: {
           name: i18n.t('js.grid.widgets.project_status_beta.title'),
         },
      },
```